dbt/adapters/athena/connections.py

```diff
@@ -56,6 +56,7 @@ class AthenaCredentials(Credentials):
     region_name: str
     endpoint_url: Optional[str] = None
     work_group: Optional[str] = None
+    work_group_enforced: Optional[bool] = None
```

Comment:
as suggested here, I proposed a better name for this parameter https://github.com/dbt-athena/dbt-athena/pull/713/files#r1746688402

Reply:
Renamed to `skip_workgroup_check`.